cmd/virt-handler/virt-handler.go

```diff
@@ -180,19 +180,6 @@ func (app *virtHandlerApp) Run() {
 		cache.Indexers{},
 	)
 
-	nodeLabellerConfigMapSharedInformer := cache.NewSharedIndexInformer(
```

Comment:
We will have to keep this informer. Otherwise we can't react to it properly (for instance deletes).